lib/astutil.js

```diff
@@ -103,6 +103,10 @@ ast.Tex.defineVisitor("contains_func", {
         // certain specific forms; see the tex_contains method above.
         return r.tex_contains_func(target);
     },
+    PAREN: function(target, f, a) {
+        // {\f@{a}}
```

Comment:
make a comment similar to line 111. people usually don't have a clue what @ stands for